.make/doc.make

```diff
@@ -0,0 +1,22 @@
+# Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+docs-gen:
```

Comment:
@htdvisser what's the plan for development tooling; shouldn't we require new stuff to be in mage?

Reply:
I agree